docs/framework/network-programming/how-to-configure-network-tracing.md

```diff
@@ -63,7 +63,8 @@ The application or computer configuration file holds the settings that determine
         <sharedListeners>  
           <add name="System.Net"  
             type="System.Diagnostics.TextWriterTraceListener"  
-            initializeData="network.log"  
+            initializeData="network.log"
+            traceOutputOptions = "ProcessId, DateTime" 
```

Comment:
To be consistent with the remaining attributes of \<add>, could you remove the spaces that precede and follow the equals sign?